src/main/java/edu/harvard/iq/dataverse/FilePage.java

```diff
@@ -959,6 +962,20 @@ public String getPublicDownloadUrl() {
         return toolsWithPreviews;
     }
     
+    public List<ExternalTool> getFileRequestAccessTools() {
+        return fileRequestAccessTools;
+    }
+    
+    public ExternalTool getFileRequestAccessTool(){
```

Comment:
Looks like the singular version is only used in the .xhtml to be able to do a null check for rendering? If so, some of the xhtml files already just check whether the list is empty (versus the first element being null). Could do that everywhere and drop these methods?